app/v4/models/request/createForeignPropertyPeriodSummary/foreignFhlEea/ForeignFhlEeaIncome.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v4.models.request.createForeignPropertyPeriodSummary.foreignFhlEea
+
+import play.api.libs.json.{Format, Json}
+
+case class ForeignFhlEeaIncome(rentAmount: Option[BigDecimal])
+
+object ForeignFhlEeaIncome {
+  implicit val format: Format[ForeignFhlEeaIncome] = Json.format
```

Comment:
This is slightly different to the other json format lines so just double checking, see this file for example https://github.com/hmrc/property-business-api/pull/476/files#diff-9c4fb439bc1f17e888ffd6fbe9c0ff17112b5b2dd361a5dbc360d3df9e99c767